build-system/server/new-server/transforms/utilities/lazy.ts

```diff
@@ -1,15 +1,15 @@
 export class Lazy<T> {
-  #value: T | undefined;
-  #initializer: () => T;
+  private declare initializer_: () => T;
+  private declare value_: T | undefined;
 
   constructor(initializer: () => T) {
-    this.#initializer = initializer;
+    this.initializer_ = initializer;
   }
 
   public get value(): T {
-    if (!this.#value) {
-      this.#value = this.#initializer();
+    if (!this.value_) {
```

Comment:
I know this is existing code...but isn't this a bug for some types? namely numbers where 0 is possible, strings where empty string is possible, etc. Not sure how it's used though
```suggestion
    if (this.value_ == undefined) {
```